### PR TITLE
i18n(zh-cn): translate `guides/integrations-guide/cloudflare.mdx`

### PR DESCRIPTION
**Pull request by hellolin-oi**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
  translate `guides/integrations-guide/cloudflare.mdx`

### CODE REVIEW

**Comment by hellolin-oi** [comment]
@liruifengv

File: src/content/docs/zh-cn/guides/integrations-guide/cloudflare.mdx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    build: {
+      minify: false,
+    },
+  },
+});
+```
+
+## 贡献
+
+该软件包由 Astro 核心团队维护。欢迎提交问题或 PR！
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/cloudflare.mdx:147]:
```suggestion
该软件包由 Astro 核心团队维护。欢迎提交 issue 或 PR！
```

**Review by liruifengv — APPROVED** [review]
A little change. Other LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks for the great work, and welcome to Team Docs @hellolin-oi!

**Pull request merged**